1.7/administration/high-availability/multi-region.md

```diff
@@ -0,0 +1,33 @@
+---
+post_title: Multiple Regions
+menu_order: 2
+---
+
+DC/OS supports multiple region configurations. This topic describes the setup recommendations and caveats.
+
+# Single Region Masters and Cross-Region Agents 
```

Comment:
How about add this to the header (same as multi-zone): `**Important:** A typical DC/OS cluster has all master and agent nodes in the same zone. The cost of having masters spread across zones usually outweighs the benefits. None of the following multi-AZ setups have been explicitly tested or verified by Mesosphere.`